packages/block-library/src/buttons/editor.native.scss

```diff
@@ -0,0 +1,3 @@
+.nestedButtons {
+	margin: $block-selected-child-margin + $block-selected-child-border-width + $block-selected-border-width + $block-selected-margin;
```

Comment:
Just a note. This one also should be gone in the future according to Piotr bordering approach